server/api.go

```diff
@@ -463,6 +463,23 @@ func (a *apiServer) subscribe(ctx context.Context, partition *partition,
 	req *client.SubscribeRequest, cancel chan struct{}) (
 	<-chan *client.Message, <-chan *status.Status, *status.Status) {
 
+	if req.AllowResuming {
+		if err := a.resumeStream(ctx, req.Stream, req.Partition); err != nil {
```

Comment:
does `resumeStream` error if the stream is not paused?

Reply:
I imagine we'll always set the AllowResuming flag

Reply:
> does `resumeStream` error if the stream is not paused?

No it returns `nil` in that case.

> I imagine we'll always set the AllowResuming flag

You mean that in go-liftbridge it should be true by default?